src/project_setup.jl

```diff
@@ -7,6 +7,16 @@ export findproject, quickactivate
 
 @deprecate scriptdir scriptsdir #TODO: remove in next release
 
+"""
+    function is_standard_julia_project()
+
+Returns true if a project other than the standard Julia project is active.
```

Comment:
I think it returns `true` if the standarrd Julia project is active, not other than the standard.

Reply:
Yeah sure! It was a bit late 😉